<ui version="4.0" >
 <class>BreakpointsWidget</class>
 <widget class="QWidget" name="BreakpointsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Breakpoints</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item rowspan="2" row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QListView" name="listView" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="minimumSize" >
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QPushButton" name="pushAdd" >
          <property name="text" >
           <string>&amp;Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushRemove" >
          <property name="text" >
           <string>&amp;Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushEnable" >
          <property name="text" >
           <string>&amp;Enable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushDisable" >
          <property name="text" >
           <string>&amp;Disable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushToggle" >
          <property name="text" >
           <string>&amp;Toggle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushEdit" >
          <property name="text" >
           <string>Ed&amp;it</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
